packages/components/src/Button/Button.js

```diff
@@ -0,0 +1,83 @@
+import { forwardRef } from 'react';
+import { createStyleSheet, alpha as convertRgba } from '@platzily-ui/styling';
+import { cx } from '@emotion/css';
+
+const useStyleSheet = createStyleSheet(
+  (theme, props) => {
+    return {
+      primaryButton: {
+        backgroundColor: props.color || theme.palette.secondary.main,
+        borderRadius: '5px',
+        width: '210px',
+        height: '40px',
+        textAlign: 'center',
+        fontWeight: '500',
+        fontSize: '20px',
+        color: theme.palette.primary.main,
+        '&:hover': {
+          color: theme.palette.text.light,
+          backgroundColor: convertRgba(props.color || theme.palette.secondary.main, 0.6),
+        },
+        '&:focus': {
+          backgroundColor: convertRgba(props.color || theme.palette.secondary.main, 0.6),
+        },
+        '&:disabled': {
+          backgroundColor: theme.palette.text.disabled,
+          color: theme.palette.text.secondary,
+          PointerEvents: 'none',
```

Comment:
Lets remove this pointer events none here and change the cursor to `not-allowed`